tests/data/state_estimation/1os2msr-no-angle/params.json

```diff
@@ -1,14 +1,5 @@
 {
   "calculation_method":  ["iterative_linear", "newton_raphson"],
   "rtol": 1e-8,
-  "atol": 1e-8,
-  "extra_params": {
-    "newton_raphson": {
-      "experimental_features": "enabled",
-      "fail": {
-        "reason": "Experimental feature under development",
-        "raises": "PowerGridError"
-      }
-    }
-  }
+  "atol": 1e-8
```

Comment:
i **do** think you need `"experimental_features": "enabled"`... if not, please LMK because that's an issue with the NRSE C API